extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -34,7 +31,15 @@ const MAX_RTC_CALLOUTS = 5;
 const MAX_URL_LENGTH = 16384;
 
 /** @type {boolean} */
-const ERROR_REPORTING_ENABLED = Math.random() < 0.01;
+const ERROR_REPORTING_ENABLED = getMode(window).localDev ||
+      getMode(window).test || Math.random() < 0.01;
+
+/** @typedef {{
+    urls: Array,
```

Comment:
Can we type better here?  e.g. urlls: (undefined|Array<string>)  and object def for vendors